widget/view.controller.js

```diff
@@ -68,14 +131,173 @@
     }
 
     function moveNext() {
-      executeGridPlaybook($scope.config.metadata.playbook, $scope.triggerStep);
+      loadPlaybookData($state.params.tab);
+      if ($scope.jsonToGrid) {
+        _checkTaskRecord();
+        executeGridPlaybook($scope.config.metadata.playbook, $scope.triggerStep);
+      }
+      else {
+        $scope.dummyRecordIRI = $scope.config.metadata.getSelectedRows['@id'].replace('/api/3/', '');
+        $rootScope.pendingDecisionModalOpen = true;
+        commentWebsocket();
+        executeGridPlaybook($scope.config.metadata.playbook, $scope.triggerStep);
+      }
+    }
+
+    function loadPlaybookData(tab) {
+      if (tab === 'logs') {
+        $scope.$broadcast('cs:getList');
+      }
+    }
+
+    function _checkTaskRecord() {
+      var queryBody = {
+        "logic": "AND",
+        "filters": [
+          {
+            "field": "name",
+            "operator": "eq",
+            "value": $scope.config.metadata.getSelectedRows[0].new_branch + '-' + $scope.config.metadata.getSelectedRows[0].summary,
```

Comment:
preferably put this in a variable above queryBody and add here.
var _xyz = $scope.config.metadata.getSelectedRows[0].new_branch + '-' + $scope.config.metadata.getSelectedRows[0].summary;

value: _xyz